packages/next/build/webpack-config.ts

```diff
@@ -816,6 +818,10 @@ export default async function getBaseWebpackConfig(
       resolveRequest: string
     ) => Promise<[string | null, boolean]>
   ) {
+    // Rewrite back to internals to avoid bundled into chunks after overriding in require-hook
+    if (request === 'styled-jsx') {
```

Comment:
Do we also want to handle mapping `styled-jsx/css`?

Reply:
They'll be transformed as well to `styled-jsx/style`, I appended one test case for it